fastlane/lib/fastlane/actions/git_branch.rb

```diff
@@ -24,9 +24,7 @@ def self.available_options
       end
 
       def self.output
-        [
-          ['GIT_BRANCH_ENV_VARS', 'The git branch environment variables']
```

Comment:
I'm slightly concerned about changing this behavior, and it's not clear to me why we needed this change 😮

Reply:
I think this was by-mistake...reverted this now! and this is not even related to this regression! 😉